aibolit/metrics/cognitiveC/cognitive_c.py

```diff
@@ -1,130 +1,139 @@
-from javalang.tree import IfStatement, SwitchStatement, ForStatement, WhileStatement
-from javalang.tree import DoStatement, CatchClause, BreakStatement, ContinueStatement
-from javalang.tree import TernaryExpression, BinaryOperation, MethodDeclaration, MethodInvocation
-from aibolit.utils.ast_builder import build_ast
-import javalang
-from typing import List, Any, Type
-
-increment_for: List[Type] = [
-    IfStatement,
-    SwitchStatement,
-    ForStatement,
-    WhileStatement,
-    DoStatement,
-    CatchClause,
-    BreakStatement,
-    ContinueStatement,
-    TernaryExpression,
-    BinaryOperation,
-    MethodInvocation,
+from itertools import groupby
+from aibolit.utils.ast import AST, ASTNodeType
+from aibolit.utils.java_package import JavaPackage
+from typing import List, Any
+import re
+
+increment_for: List[ASTNodeType] = [
```

Comment:
We better make it a set, if we don't get its elements by index.
Same with `nested_for`

Reply:
got it